### PR TITLE
Add user class api repo authentication authorization

### PR DESCRIPTION
**Pull request by lukaszkosminski**

### CODE REVIEW

File: my-fund-backend/src/main/java/com/myfund/controller/BudgetController.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+import com.myfund.model.DTO.BudgetDTO;
+import com.myfund.model.DTO.CreateBudgetDTO;
+import com.myfund.model.User;
+import com.myfund.service.BudgetService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.web.bind.annotation.*;
+
+@CrossOrigin(origins = "http://localhost:4200")
```
Comment (Maciekek) [inline my-fund-backend/src/main/java/com/myfund/controller/BudgetController.java:11]: Probably for later, but this values should come from some config

File: my-fund-backend/src/main/java/com/myfund/model/Budget.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+@Getter
+@Setter
+@Entity
+@ToString
+@Table(name = "budget")
+public class Budget {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
```
Comment (Maciekek) [inline my-fund-backend/src/main/java/com/myfund/model/Budget.java:16]: for model classes, it is good to add new line sign, after each property, it is easier to read then

File: my-fund-backend/src/main/java/com/myfund/model/Budget.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    private String nameBudget;
```
Comment (Maciekek) [inline my-fund-backend/src/main/java/com/myfund/model/Budget.java:17]: `name` is fine, name`Budget` in Budget is redundant

**Review by Maciekek — APPROVED** [review]

File: my-fund-backend/src/main/java/com/myfund/controller/BudgetController.java (hunk 1 above)
Comment (lukaszkosminski) [reply my-fund-backend/src/main/java/com/myfund/controller/BudgetController.java]: I moved this value to the application properties.

File: my-fund-backend/src/main/java/com/myfund/model/Budget.java (hunk 2 above)
Comment (lukaszkosminski) [reply my-fund-backend/src/main/java/com/myfund/model/Budget.java:16]: added new line after each property

File: my-fund-backend/src/main/java/com/myfund/model/Budget.java (hunk 3 above)
Comment (lukaszkosminski) [reply my-fund-backend/src/main/java/com/myfund/model/Budget.java]: variable name changed

**Pull request merged**